src/services/edusharing/hooks/index.js

```diff
@@ -0,0 +1,31 @@
+const { Configuration } = require('@schul-cloud/commons');
+const { authenticate } = require('@feathersjs/authentication');
+const { disallow } = require('feathers-hooks-common');
+const { Forbidden } = require('@feathersjs/errors');
+
+const isEdusharing = (context) => {
```

Comment:
this duplicates via https://github.com/schul-cloud/schulcloud-server/pull/965/files#diff-783e386378e392ee200db5095c8ab5d7R17

Reply:
Extracted to hook method and reused in edusharing/index

Reply:
Removed from edusharing service. Since hook manages this.